spec/models/manageiq/providers/vmware/cloud_manager/orchestration_service_option_converter_spec.rb

```diff
@@ -4,47 +4,49 @@
   let(:dialog_options) do
     {
       # 1st vapp network
-      'dialog_param_parent-0'           => nil,
-      'dialog_param_fence_mode-0'       => 'isolated',
-      'dialog_param_gateway-0-0'        => '192.168.0.0',
-      'dialog_param_netmask-0-0'        => '255.255.255.0',
-      'dialog_param_dns1-0-0'           => '8.8.8.8',
-      'dialog_param_dns2-0-0'           => nil,
+      'dialog_param_parent-0'              => nil,
+      'dialog_param_fence_mode-0'          => 'isolated',
+      'dialog_param_gateway-0-0'           => '192.168.0.0',
+      'dialog_param_netmask-0-0'           => '255.255.255.0',
+      'dialog_param_dns1-0-0'              => '8.8.8.8',
+      'dialog_param_dns2-0-0'              => nil,
       # 2nd vapp network
-      'dialog_param_parent-1'           => 'b915be99-1471-4e51-bcde-da2da791b98f',
-      'dialog_param_fence_mode-1'       => 'bridged',
-      'dialog_param_gateway-1-0'        => '192.168.0.1',
-      'dialog_param_netmask-1-0'        => '255.255.255.1',
-      'dialog_param_dns1-1-0'           => '1.2.3.4',
-      'dialog_param_dns2-1-0'           => '4.3.2.1',
+      'dialog_param_parent-1'              => 'b915be99-1471-4e51-bcde-da2da791b98f',
+      'dialog_param_fence_mode-1'          => 'bridged',
+      'dialog_param_gateway-1-0'           => '192.168.0.1',
+      'dialog_param_netmask-1-0'           => '255.255.255.1',
+      'dialog_param_dns1-1-0'              => '1.2.3.4',
+      'dialog_param_dns2-1-0'              => '4.3.2.1',
       # 1st VM
-      'dialog_param_instance_name-0'    => 'my VM1',
-      'dialog_param_hostname-0'         => 'my-vm-1',
-      'dialog_param_num_cores-0'        => 8,
-      'dialog_param_cores_per_socket-0' => 4,
-      'dialog_param_memory_mb-0'        => 8192,
-      'dialog_param_admin_password-0'   => 'admin-password',
-      'dialog_param_admin_reset-0'      => 't',
-      'dialog_param_disk_mb-0-0'        => 40_960,
-      'dialog_param_disk_mb-0-1'        => 20_480,
-      'dialog_param_nic_network-0-0'    => 'VM Network',
-      'dialog_param_nic_mode-0-0'       => 'MANUAL',
-      'dialog_param_nic_ip_address-0-0' => '192.168.0.100',
+      'dialog_param_instance_name-0'       => 'my VM1',
+      'dialog_param_hostname-0'            => 'my-vm-1',
+      'dialog_param_num_cores-0'           => 8,
+      'dialog_param_cores_per_socket-0'    => 4,
+      'dialog_param_memory_mb-0'           => 8192,
+      'dialog_param_admin_password-0'      => 'admin-password',
+      'dialog_param_admin_reset-0'         => 't',
+      'dialog_param_guest_customization-0' => 't',
```

Comment:
So you've introduced this here in the input (which is BTW a simultaion what we get throught POST request when user orders provisioning). Can you please set it to `'f'` in at least one case and check later in this test (L101 or L129)?